draft-ietf-mls-protocol.md

```diff
@@ -1910,46 +1910,45 @@ The client verifies the validity of a LeafNode using the following steps:
   another LeafNode, the authentication service MUST additionally validate that
   the set of identities attested by the credential in the new LeafNode is
   acceptable relative to the identities attested by the old credential.
+  For example:
+    * An Update proposal updates the sender's old LeafNode to a new one
+    * A "resync" external commit removes the joiner's old LeafNode via a Remove proposal and replaces it with a new one
```

Comment:
Also checking if identities are the same in a resync is a good point, but isn't it undermined by doing the external join and remove separately?

Reply:
I think this is covered in the External Commits section, but mentioning it here or including a reference probably doesn't hurt.

Reply:
Currently this requirement is in the External Commit section:
>  There MAY be a single Remove proposal, where the LeafNode in the path field MUST meet the same criteria as the LeafNode in an Update for the removed leaf (see Section 13.1.2). In particular, the credential in the LeafNode MUST present a set of identifiers that is acceptable to the application for the removed participant.

The PR moves it here (so it's only in one place).

This PR is editorial, so I'd rather not remove the restriction here but instead discuss this as a separate issue. I've also been wondering why only one Remove is allowed? E.g. I may want to remove a couple of my old devices.

Reply:
Agree that we should handle the revalidation separately, in #660.

On "why only one" -- The idea is to *only* cover the resync case, where a device is replacing *its own* state.  Other devices can be cleaned up after the new member has joined.